Header header
int32 number_of_detections
Detection[] array
